github_client/github.go

```diff
@@ -1,11 +1,12 @@
-package github
+package github_client
```

Comment:
[Lines 465-514] Potential Bug: TestDiffTransformer_numberLines has been removed without any explanation. Was this test merged elsewhere or should it be included in this PR? Please clarify.